receiver/faroreceiver/metadata.yaml

```diff
@@ -0,0 +1,13 @@
+type: faro
+
+status:
+  class: receiver
+  stability:
+    development: [logs, traces]
+  distributions: [contrib]
```

Comment:
Don't add to contrib until it's complete.

Reply:
sorry yes my bad, I just copied this across from the other pr